pkg/rpc/server.go

```diff
@@ -311,6 +315,26 @@ func (s *Server) getrawtransaction(reqParams Params) (interface{}, error) {
 	return results, resultsErr
 }
 
+// getContractState returns contract state (contract information, according to the contract script hash).
+func (s *Server) getContractState(reqParams Params) (interface{}, error) {
+	var results interface{}
+
+	param, ok := reqParams.ValueWithType(0, stringT)
+	if !ok {
+		return nil, errInvalidParams
+	} else if scriptHash, err := param.GetUint160FromHex(); err != nil {
+		return nil, errInvalidParams
+	} else {
+		cs := s.chain.GetContractState(scriptHash)
+		if cs != nil {
+			results = wrappers.NewContractState(cs)
+		} else {
+			results = "Unknown contract"
```

Comment:
Is it returned as an error or as a result in C# node?

Reply:
It is returned as en error, done